cdk-ts/import-service/lambda/importFileParser.ts

```diff
@@ -0,0 +1,64 @@
+import { CopyObjectCommand, DeleteObjectCommand ,GetObjectCommand, S3Client } from "@aws-sdk/client-s3";
+import * as csv from "csv-parser";
+import { Readable } from "stream";
+
+const client = new S3Client({});
+
+export const setResponse = (
+    statusCode: number = 200,
+    body = {},
+) => {
+    return {
+        statusCode,
+        headers: {
+            "Access-Control-Allow-Origin": "*",
+            "Access-Control-Allow-Headers": "*",
+            "Access-Control-Allow-Methods": "*",
+            "Content-Type": "application/json",
+        },
+        body: body,
+    };
+};
+
+export const handler = async (event: any) => {
+    console.log('Upload file with products list event', event);
+
+    const bucketName = event.Records[0].s3.bucket.name;
```

Comment:
What will happen in the case if records number is > 1?

Reply:
Thanks, fixed